maps/torch/items/summarydocuments.dm

```diff
@@ -208,160 +192,137 @@
+	desc = "Some \"internal use only\" reports about personnel smoking near the atmospherics holding area aboard the SEV Charon. The atmospherics holding area aboard the Charon is declared a no-smoking area, as indicated by the sign. Attached is a brief incident report with a contracted pilot. The report ends with large-font writing stating the next person found smoking near the fueling port will be demoted to sanitation tech for the rest of their deployment."
 
 /obj/item/documents/generic/explo/survivors
 	name = "interactions with survivors"
-	desc = "Some \"internal use only\" reports discussing the process of dealing with survivors, off-ship personnel or in-distress civilians. This one details three points that all Explorers, Fleet or contractor personnel should attempt to follow when dealing with off-ship persons: First, treat them with the utmost respect at all times, as they are the ones who need aid. Second, do not directly order them around, unless they happen to be getting in the way of SCG objectives. Third, remind them of their rights - they are requiring aid and are to be treated with the best handling, but remind them that they are still on SCG property, and are subject to SCG law at all times."
-
-/obj/item/document/generic/explo/holocalletiquette
-	name = "holocall etiquette"
-	desc = "Some \"internal use only\" reports, containing a small guide on proper use of holocalls from the perspective of the SEV Charon. This one details five points. First, Calls are a one-way interaction. You inform them of the basic situation, they inform you of what they can do to help. Second, Calls are two-way if you request them to be. Third, If an executive decision is required, ask the Senior on the ground. If he does not render a decision, the SEV Torch shall, but with an emphasis on being decisive. Fourth, State aloud when the call connects. It helps cut down on background chatter. Fifth, The SEV Torch doesn't need to be informed on every specific thing Exploration is doing while on Holocall."
-
-/obj/item/document/generic/explo/holocallloiter
-	name = "holocall etiquette"
-	desc = "Some \"internal use only\" reports, requesting people that if a holocall lingers, they should make a point to end it."
+	desc = "Some \"internal use only\" reports discussing the process of dealing with survivors, off-ship personnel, or in-distress civilians. This one details three points that all explorers, Fleet servicemembers, or contractor personnel should attempt to follow when dealing with off-ship persons: First, treat them with the utmost respect at all times, as they are the ones who need aid. Second, do not directly order them around, unless they happen to be getting in the way of SCG objectives. Third, remind them of their rights - they are requiring aid and are to be treated with the best handling, but remind them that they are still on SCG property, and are subject to SCG law at all times."
```

Comment:
I'm personally still iffy on this one since it seems too prescriptive about handling foreign people, but it seems alright as-is.

Reply:
Bit too wordy, but workable.